workers/ohsome_quality_analyst/oqt.py

```diff
@@ -117,37 +246,9 @@ async def create_all_indicators(force: bool = False) -> None:
                     raise (error)
 
 
-async def create_report(
-    report_name: str,
-    force: bool = False,
-    feature: Optional[Feature] = None,
-    dataset: Optional[str] = None,
-    feature_id: Optional[str] = None,
-    fid_field: Optional[str] = None,
-) -> BaseReport:
-    """Create a report.
-
-    A Reports creates indicators from scratch or fetches them from the database.
-    It then aggregates all indicators and calculates an overall quality score.
-    """
-    if feature is None and dataset is not None and feature_id is not None:
-        feature = await db_client.get_feature_from_db(dataset, feature_id, fid_field)
-
-    report_class = name_to_class(class_type="report", name=report_name)
-    report = report_class(
-        feature=feature, dataset=dataset, feature_id=feature_id, fid_field=fid_field
-    )
-    report.set_indicator_layer()
-    for indicator_name, layer_name in report.indicator_layer:
-        indicator = await create_indicator(
-            indicator_name,
-            layer_name,
-            feature=report.feature,
-            dataset=report.dataset,
-            feature_id=report.feature_id,
-            fid_field=fid_field,
-            force=force,
+async def check_area_size(geom: Union[Polygon, MultiPolygon]):
+    if await db_client.get_area_of_bpolys(geom) > GEOM_SIZE_LIMIT:
+        raise ValueError(
+            "Input GeoJSON Object is too big. "
+            + "The area should be less than {0} sqkm.".format(GEOM_SIZE_LIMIT)
```

Comment:
```suggestion
            "The area should be less than {0} sqkm.".format(GEOM_SIZE_LIMIT)
```

Reply:
Another suggestion:
```suggestion
            "The area should be less than {0} km².".format(GEOM_SIZE_LIMIT)
```

Reply:
I changed the formatting to you second suggestion.